app_server.py

```diff
@@ -349,7 +320,7 @@ def work_lookup(self, annotator, route_name='lookup', **process_urn_kwargs):
             self._db, "Lookup results", this_url, handler.works, annotator,
             precomposed_entries=handler.precomposed_entries,
         )
-        return feed_response(opds_feed)
+        return OPDSFeedResponselike(opds_feed).response
```

Comment:
In a future branch we could change these to return a Responselike, but since this isn't the main focus of the branch I decided to go through ResponseLike to Response and leave the contract -- that you call these functions and get a Response -- unaltered.